vnext/Microsoft.ReactNative/JsiApi.cpp

```diff
@@ -0,0 +1,432 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JsiApi.h"
+#include "JsiPreparedJavaScript.g.cpp"
+#include <crash/verifyElseCrash.h>
+
+// facebook::jsi::Runtime hides all methods that we need to call.
+// We "open" them up here by redeclaring the private and protected keywords.
+#define private public
```

Comment:
>private public [](start = 8, length = 14)

Out of curiosity, could this mess with TMP, e.g. by altering SFINAE?

Reply:
No, it should not. The change is local for this .cpp only. I do not expect any conflicts with other pats of the code. In future we may find a better way to achieve the same.

---
In reply to: [451853914](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r451853914) [](ancestors = 451853914)